README.md

```diff
@@ -27,4 +28,6 @@ The API can be tested with postman as described below:
   - fileContent: It is the file itself
 - Search for words by making another request to localhost:3000/api/search with the following form-data bodies:
   - fileName (Optional): The document you want to search from
-  - searchTerms: The words you are searching. It can be an array of words.
\ No newline at end of file
+  - searchTerms: The words you are searching. It can be an array of words.
+  
+Note: [-1] denotes 'word not found'
```

Comment:
Well done so far with the content of your README file, however, to better the user of your application better understand how to navigate and use this application, you might want to go a bit into details on how to install, clone, contribute, and dependencies used admist others.

Also, you might want to also put an example file of what a valid file content should look like and examples of the result to expect from the two API endpoints when hit.